setup/account_credit_control_analysis/setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
+
+setuptools.setup(
+    setup_requires=['setuptools-odoo'],
+    odoo_addon=True,
+)
```

Comment:
@qgroulard these setup.py now refers to an non-existent module. Can you remove it?

After that we can merge.